### ISSUE TITLE
Pausing script (F10) makes cursor unmovable with mouse

### ISSUE THREAD

**Issue by nikalsh** [bennybroseph/AutoHotKey_Scripts#8]
- Running in admin mode
- Controller disabled
- Mouse & Keyboard enabled
- Path of Exile profile
- Path of Exile keybinds

Pausing the script with F10 disables the script as expected but makes the cursor unmovable with the actual mouse. Current workaround is to completely terminate the scripts with F12, do regular mouse + keyboard stuff then start the script again.

**Comment by bennybroseph** [comment]
Yeah, I think the way I was getting this to work at the time was creating a completely separate script that runs in the background to interrupt mouse movements. Since it's not on the main thread and communication was only opened one way, it doesn't know that it needs to pause. Maybe there is a new way to go about this since AHK has updated since I last worked on this. Just commenting for documentation/historical purposes.
